libseccomp/src/lib.rs

```diff
@@ -1086,6 +1059,28 @@ mod tests {
         assert_eq!(expected_api, api);
     }
 
+    #[test]
+    fn test_scmpargcompare() {
```

Comment:
Could you add or replace tests for `From` trait as follows? ( currently, the trait is not covered by tests)

```rust
 assert_eq!(                                                                 
     scmp_arg_cmp::from(ScmpArgCompare::new(0, ScmpCompareOp::NotEqual, 8)), 
     scmp_arg_cmp {                                                          
         arg: 0,                                                             
         op: scmp_compare::SCMP_CMP_NE,                                      
         datum_a: 8,                                                         
         datum_b: 0,                                                         
     }                                                                       
 );                                                                          
 assert_eq!(                                                                 
     scmp_arg_cmp::from(&ScmpArgCompare::new(0, ScmpCompareOp::NotEqual, 8)),
     scmp_arg_cmp {                                                          
         arg: 0,                                                             
         op: scmp_compare::SCMP_CMP_NE,                                      
         datum_a: 8,                                                         
         datum_b: 0                                                          
     }                                                                       
 );
```